public/data/abhishinde24.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "Abhishek Shinde",
+    "type": "personal",
+    "bio": "Machine Learning, Data Science, DSA Entusiast",
+    "avatar": "https://avatars.githubusercontent.com/abhishinde24",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/abhishinde24",
+        "icon": "github"
+      },
+      {
+        "name": "Connect with me on LinkedIn",
+        "url": "https://www.linkedin.com/in/abhishek-shinde-0a1860192",
+        "icon": "linkedin"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started DSA,CP,Machine learing,",
+        "date": "June 2021",
+        "icon": "code",
+        "color": "#90ee90",
+        "description": "Began Practicing Data structures and Algorithm, Competitive programing and Machine learning side projects",
+        "url": "#"
```

Comment:
Kindly add your URL here, If you don't have a URL you can simply remove the line no `25`

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix